src/assets/scss/application.scss

```diff
@@ -91,3 +91,7 @@ $govuk-assets-path: "/public/";
   margin-top: 0; 
   margin-bottom: 20px;
 }
+
+.govuk-details {
```

Comment:
Is this class being used anywhere?

Reply:
Yes thats for the dropdown with the new instruction in, AC3 called for a change to the custom spacing between the detail component and the continue button

Reply:
Ah ok - reckon we definitely need a custom class or can we use the gov design system one again?